app/scripts/directives/mainSearchBoxHeight.dir.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('yourStlCourts').directive('mainSearchBox', function ($window) {
+  return {
+    restrict: 'E',
+    replace: true,
+    transclude: true,
+    template: '<div id="mainSearchBox" class="accent" ng-transclude></div>',
+    scope: false,
+    link: function ($scope, element, attrs) {
+      angular.element($window).on('resize',function(){
+        resizeMainSearchBox();
+      });
+      resizeMainSearchBox(); //call it for initial setup
+
+      function resizeMainSearchBox(){
+        var newHeight = $(element).parentsUntil('body').parent().width()*.3125;
```

Comment:
$window.innerWidth gives different results using different browsers.  using JQuery's .width() normalizes this.  Using the <body> tag is ok for this usage, since the image being scaled is at 100% width of the body.